### PR TITLE
Feature/i18n

### PR DESCRIPTION
**Pull request by wilfriedwolf**
Hi Simon,

this PR allows to use different Settings for different StoreViews and especially adds configurable data-culture attribute, so the message can be in different languages according to the StoreView settings.

Best Regards,

Willi

### CODE REVIEW

File: etc/adminhtml/system.xml [hunk 1]
```diff
@@ -15,6 +15,12 @@
                         <field id="enabled">1</field>
                     </depends>
                 </field>
+                <field id="data_culture" translate="label comment" type="text" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
```
Comment (sprankhub) [inline etc/adminhtml/system.xml:18]: I think this should be a `select` field with all possible "two-letter ISO language codes" (https://support.cookiebot.com/hc/en-us/articles/360003816053-Make-CMS-control-language).

File: etc/config.xml [hunk 2]
```diff
@@ -6,6 +6,7 @@
             <cookiebot>
                 <enabled>0</enabled>
                 <id/>
+                <data_culture />
```
Comment (sprankhub) [inline etc/config.xml:9]: Please remove the whitespace :see_no_evil:

**Review by sprankhub — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution! I left two improvement suggestions. Are you willing to add these? If not, I am happy to do it.

**Comment by wilfriedwolf** [comment]
I could certainly add the improvements. Just give me a hint for a source for the "two-letter ISO language codes"... Is there a source class in M2?

**Comment by sprankhub** [comment]
I do not think there is an existing source model for this. But I think it should be easily possible to retrieve the correct data from `directory_country.iso2_code`. I am not totally sure if this is the right code Cookiebot expects, but I guess so :)

**Comment by wilfriedwolf** [comment]
Well, actually I do not think so, I was looking there to, but these are country codes, not language code, there is no 'AT' language, although some think it is different (well would be in 'de_AT')...

I was digging a little deeper: https://www.w3.org/International/questions/qa-lang-2or3 states that https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry is the official source.

Another way would be, to use the intl extension and parsing, like 

```
 $b = new \ResourceBundle('de_DE','ICUDATA');
 print_r( $b->getLocales(''));
 ```
 And then using the toupper 2 letter part in front of '_'
 
 Anyway i am questioning the use of a pulldown here, since you can just validate 2 upper case letters and you have to check anyway if cookiebot supports the desired language, which for example 'AZ' I try to doubt.

File: etc/adminhtml/system.xml (hunk 1 above)
Comment (sprankhub) [reply etc/adminhtml/system.xml:18]: I agree that we should keep it as it is.

File: etc/config.xml (hunk 2 above)
Comment (sprankhub) [reply etc/config.xml:9]: Will be fixed with a separate commit.

**Review by sprankhub — APPROVED** [review]

**Pull request merged**